File: Fizzbuzz.java
// Fizzbuzz

public class Fizzbuzz {

  public static void main(String[] args) {
    for(int x = 1; x <= 100; x++) {
      if (x % 3 == 0 && x % 5 == 0) {
        System.out.print("Fizzbuzz\n");
      } else if (x % 3 == 0) {
        System.out.print("Fizz\n");
      } else if (x % 5 == 0) {
        System.out.print("Buzz\n");
      } else {
        System.out.print(x + "\n");
      }
    }
  }
}

File: fizzbuzz.pl
# Fizzbuzz

# Strict and warnings are recommended.
use strict;
use warnings;

for( $a = 1; $a <= 100; $a++ ){
  if ($a % 3 == 0 && $a % 5 == 0) {
    print "FizzBuzz\n";
  } elsif ($a % 3 == 0) {
    print "Fizz\n";
  } elsif ($a % 5 == 0) {
    print "Buzz\n";
  } else {
    print "$a\n";
  }
}
